### PR TITLE
Daylight: Cards

### PR DESCRIPTION
**Pull request by GeorgePlukov**

### CODE REVIEW

File: components/card/README.md [hunk 1]
```diff
@@ -1,30 +1,132 @@
-# Card
+# Cards
+Cards provide concise information and actions as they relate to a single object. They contain content such as images, text, lists, data, rich media, actions, and more.
 
-## d2l-card
+Used to surface pertinent information, cards make it easy for users to browse information.
+<!-- docs: demo -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/card/card.js';
```
Comment (margaree) [inline components/card/README.md:8]: nit: order of imports could have card imports together

File: components/card/README.md [hunk 2]
```diff
@@ -1,30 +1,132 @@
...
+
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Roll up and emphasize important information to make it easy for users to find
+* Use cards when the user will benefit from a visual representation of the associated item
+* Use cards when the content doesn’t rely heavily on the sort order
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Force the user to click in and out of cards to find information that’s important to the work flow
```
Comment (margaree) [inline components/card/README.md:38]: These should start with "Don't"

File: components/card/README.md [hunk 3]
```diff
@@ -1,30 +1,132 @@
-# Card
+# Cards
```
Comment (margaree) [inline components/card/README.md:1]: nit: we usually have a line between headings and content (also noticed below)

File: components/card/README.md [hunk 4]
```diff
@@ -1,30 +1,132 @@
...
+  <div slot="footer">
+    <d2l-card-footer-link id="discussionsLink1" icon="tier1:outcomes" text="Outcomes" secondary-text="2"></d2l-card-footer-link>
+    <d2l-tooltip for="discussionsLink1">Discussions</d2l-tooltip>
+    <d2l-card-footer-link id="assignmentsLink1" icon="tier1:assignments" text="Assignments" secondary-text="1"></d2l-card-footer-link>
+    <d2l-tooltip position="top" style="width: 100%;" for="assignmentsLink1">You have 1 assignments due tomorrow.</d2l-tooltip>
+  </div>
+</d2l-card>
+```
+
+### Best Practices
```
Comment (margaree) [inline components/card/README.md:28]: This should be h2

File: components/card/README.md [hunk 5]
```diff
@@ -1,30 +1,132 @@
...
+
+<!-- docs: start donts -->
+* Force the user to click in and out of cards to find information that’s important to the work flow
+* Use cards when the user needs to easily compare data from one card to another
+* Use cards when the sort order needs to be emphasized – consider a list or table
+* Use cards for user generated content
+<!-- docs: end donts -->
+<!-- docs: end best practices -->
+
+### Composition
```
Comment (margaree) [inline components/card/README.md:45]: Also should be h2

File: components/card/README.md [hunk 6]
```diff
@@ -82,44 +193,66 @@ The `d2l-card-footer-link` element is an icon link that can be placed in the `fo
...
 
 See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
+<!-- docs: end hidden content -->
 
-## d2l-card-content-title
+## Card content Title <d2l-card-content-title>
 
 The `d2l-card-content-title` element is a helper for providing layout/style for a title within the `content` slot.
 
+<!-- docs: demo live name:d2l-card-footer-link size:large -->
```
Comment (margaree) [inline components/card/README.md:202]: Should this demo be for `d2l-card-content-title`?

File: components/card/README.md [hunk 7]
```diff
@@ -82,44 +193,66 @@ The `d2l-card-footer-link` element is an icon link that can be placed in the `fo
 | `target` | String | Where to display the linked URL |
 
 See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
+<!-- docs: end hidden content -->
 
-## d2l-card-content-title
+## Card content Title <d2l-card-content-title>
```
Comment (margaree) [inline components/card/README.md:198]:
```suggestion
## Card Content Title <d2l-card-content-title>
```

File: components/card/README.md [hunk 8]
```diff
@@ -82,44 +193,66 @@ The `d2l-card-footer-link` element is an icon link that can be placed in the `fo
...
+    <d2l-card-footer-link id="discussionsLink1" icon="tier1:outcomes" text="Outcomes" secondary-text="2"></d2l-card-footer-link>
+    <d2l-tooltip for="discussionsLink1">Discussions</d2l-tooltip>
+    <d2l-card-footer-link id="assignmentsLink1" icon="tier1:assignments" text="Assignments" secondary-text="1"></d2l-card-footer-link>
+    <d2l-tooltip position="top" style="width: 100%;" for="assignmentsLink1">You have 1 assignments due tomorrow.</d2l-tooltip>
   </div>
 </d2l-card>
 ```
 
-## d2l-card-content-meta
+## Card content meta <d2l-card-content-meta>
```
Comment (margaree) [inline components/card/README.md:225]:
```suggestion
## Meta Card Content <d2l-card-content-meta>
```

File: components/card/README.md (hunk 7 above)
Comment (margaree) [reply components/card/README.md:198]: Maybe all these card contents should be something like `Card Content: Title`, `Card Content: Footer` etc

File: components/card/README.md [hunk 9]
```diff
@@ -47,28 +154,11 @@ The `d2l-card` element is a container that provides specific layout using severa
...
-  </div>
-  <div slot="footer">
-    <d2l-card-footer-link icon="..." text="..." secondary-text="..." href="..."></d2l-card-footer-link>
-  </div>
-</d2l-card>
-```
-
-**Properties:**
+<!-- docs: start hidden content -->
+### Properties:
```
Comment (margaree) [inline components/card/README.md:161]: I think these were also supposed to be moved with the rest of the `d2l-card-footer-link` section

File: components/card/README.md [hunk 10]
```diff
@@ -82,44 +172,93 @@ The `d2l-card-footer-link` element is an icon link that can be placed in the `fo
...
 
 See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
+<!-- docs: end hidden content -->
 
-## d2l-card-content-title
+## Card content: Title <d2l-card-content-title>
 
 The `d2l-card-content-title` element is a helper for providing layout/style for a title within the `content` slot.
 
+<!-- docs: demo live name:d2l-card-content-title size:large -->
```
[the remark this answers is not in the document]
Comment (GeorgePlukov) [reply components/card/README.md:163]: Fixed! This component has no properties so harder to notice but still a facepalm

[the remark this answers is not in the document]
Comment (margaree) [reply components/card/README.md:163]: No worries! One more thing here - I think `d2l-card-content-title` also needs to be imported

File: components/card/README.md [hunk 11]
```diff
@@ -47,79 +154,107 @@ The `d2l-card` element is a container that provides specific layout using severa
...
-| `href` | String | Location for the primary action/navigation |
-| `rel` | String | Relationship of the target object to the link object |
-| `secondary-text` | String | Text to display as a superscript on the icon |
-| `secondary-text-type` | String | Controls the style of the secondary text bubble; options are `notification` and `count` |
-| `target` | String | Where to display the linked URL |
-
-See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
-
-## d2l-card-content-title
+## Card content: Meta  <d2l-card-content-meta>
```
Comment (margaree) [inline components/card/README.md:185]:
```suggestion
## Card Content: Meta  [d2l-card-content-meta]
```

File: components/card/README.md [hunk 12]
```diff
@@ -47,79 +154,107 @@ The `d2l-card` element is a container that provides specific layout using severa
 | `text` | String | Accessible text for the card (will be announced when AT user focuses) |
 
 See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
+<!-- docs: end hidden content -->
 
-## d2l-card-footer-link
+## Card content: Title <d2l-card-content-title>
```
Comment (margaree) [inline components/card/README.md:159]:
```suggestion
## Card Content: Title [d2l-card-content-title]
```

File: components/card/README.md [hunk 13]
```diff
@@ -1,30 +1,137 @@
...
+      <d2l-card-footer-link id="discussionsLink1" icon="tier1:outcomes" text="Outcomes" secondary-text="2"></d2l-card-footer-link>
+      <d2l-tooltip for="discussionsLink1">Discussions</d2l-tooltip>
+      <d2l-card-footer-link id="assignmentsLink1" icon="tier1:assignments" text="Assignments" secondary-text="1"></d2l-card-footer-link>
+      <d2l-tooltip position="top" style="width: 100%;" for="assignmentsLink1">You have 1 assignments due tomorrow.</d2l-tooltip>
+    </div>
+  </d2l-card>
+</div>
+```
+
+## Card <d2l-card>
```
Comment (margaree) [inline components/card/README.md:102]: Just noticed the `<>` around the tags - I think they still have to be square brackets instead

File: components/card/README.md [hunk 14]
```diff
@@ -47,79 +154,107 @@ The `d2l-card` element is a container that provides specific layout using severa
...
+    <d2l-card-footer-link id="discussionsLink1" icon="tier1:outcomes" text="Outcomes" secondary-text="2"></d2l-card-footer-link>
+    <d2l-tooltip for="discussionsLink1">Discussions</d2l-tooltip>
+    <d2l-card-footer-link id="assignmentsLink1" icon="tier1:assignments" text="Assignments" secondary-text="1"></d2l-card-footer-link>
+    <d2l-tooltip position="top" style="width: 100%;" for="assignmentsLink1">You have 1 assignments due tomorrow.</d2l-tooltip>
   </div>
 </d2l-card>
 ```
 
-## d2l-card-content-meta
+## Card Footer: Link <d2l-card-footer-link>
```
Comment (margaree) [inline components/card/README.md:212]: This one can probably just be `Card Footer Link` - can just see what Jeff thinks on review too

**Review by margaree — APPROVED** [review]

**Pull request merged**